<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutSettingWidget">
   <property name="spacing">
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="pbRatio">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridSetting">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupAccessory">
       <property name="title">
        <string>악세</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="vLayoutAccessory"/>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupAbility">
       <property name="title">
        <string>특성</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="vLayoutAbility"/>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupSetEffect">
       <property name="title">
        <string>세트 효과</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="vLayoutSetEffect"/>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupClassEngrave">
       <property name="title">
        <string>직업 각인</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="hLayoutClassEngrave"/>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupEngrave">
       <property name="title">
        <string>각인</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="hLayoutEngrave"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
